explorer/explorerroutes.go

```diff
@@ -140,30 +143,90 @@ func (exp *explorerUI) DisapprovedBlocks(w http.ResponseWriter, r *http.Request)
 	io.WriteString(w, str)
 }
 
+// show only regular tx in block.Transactions, exclude coinbase (reward) transactions
+// for use in NextHome handler and websocket response to getmempooltxs event
+func filterRegularTx(txs []*TrimmedTxInfo) []*TrimmedTxInfo {
+	transactions := make([]*TrimmedTxInfo, 0)
+	for _, tx := range txs {
+		if !tx.Coinbase {
+			transactions = append(transactions, tx)
+		}
+	}
+	return transactions
+}
+
 // NextHome is the page handler for the "/nexthome" path.
 func (exp *explorerUI) NextHome(w http.ResponseWriter, r *http.Request) {
+	// Get top N blocks and trim each block to have just the fields required for this page.
 	height := exp.blockData.GetHeight()
+	blocks := exp.blockData.GetExplorerFullBlocks(height, height-homePageBlocksMaxCount)
+
+	// trim unwanted data in each block
+	trimmedBlocks := make([]*TrimmedBlockInfo, 0, len(blocks))
+	for index := range blocks {
```

Comment:
Since it is a slice of pointers, best to use the second range value (a copy of the element):

```go
for _, block := range blocks {
```

and omit `block := blocks[index]` of course.